bitbucket-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+# Define the Docker image to use for all steps
+image: 
+  name: aaronmedinaaligent/shopify-cli-base:latest
```

Comment:
Same as above, will need to be public or publicly accessible.

Reply:
Will update this